<ui version="4.0" >
 <class>AnimationPage</class>
 <widget class="QWidget" name="AnimationPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Animation</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="labelScroll" >
     <property name="text" >
      <string>Color Scrolling</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="DoubleSlider" name="sliderScrolling" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum" >
        <number>-100</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval" >
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopScrolling" >
       <property name="text" >
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelAuto" >
     <property name="text" >
      <string>Mesh Rotation</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="DoubleSlider" name="sliderRotation" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum" >
        <number>-100</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval" >
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopRotation" >
       <property name="text" >
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSlider</class>
   <extends>QSlider</extends>
   <header>doubleslider.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sliderScrolling</tabstop>
  <tabstop>stopScrolling</tabstop>
  <tabstop>sliderRotation</tabstop>
  <tabstop>stopRotation</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
